### PR TITLE
Use efficient bitwise operation 

### PR DESCRIPTION
**Pull request by resorcap**

### CODE REVIEW

File: Number of 1 Bits.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Solution:
+    def hammingWeight(self, n):
+        count = 0
+        while n:
+            count += 1;
```
Comment (jw2013) [inline Number of 1 Bits.py]: nit: remove ;

File: Number of 1 Bits.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+            n = (n-1)&n
```
Comment (jw2013) [inline Number of 1 Bits.py]: nit: (n - 1) & n

**Comment by jw2013** [comment]
would you please remove `;` on line 5 and add some space around parentheses on line 6 (change it to `(n - 1)`).

Otherwise it looks good to me.

**Comment by SocialfiPanda** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**